gapic/samplegen/samplegen.py

```diff
@@ -13,29 +13,50 @@
 # limitations under the License.
 
 import itertools
+import jinja2
 import keyword
 import re
-import gapic.samplegen.utils
+
+from gapic.samplegen import utils
 
 from collections import (defaultdict, namedtuple)
-from typing import (List, Mapping, Set)
+from typing import (List, Mapping, Set, Tuple)
 
 # Outstanding issues:
-# * Neither license nor copyright are defined
 # * In real sample configs, many variables are
 #   defined with an _implicit_ $resp variable.
 
 MIN_SCHEMA_VERSION = (1, 2, 0)
 
 VALID_CONFIG_TYPE = 'com.google.api.codegen.SampleConfigProto'
 
-# There are a couple of other names that should be reserved
-# for sample variable assignments, e.g. 'client', but there are diminishing returns:
-# the sample config author isn't a pathological adversary, they're a normal person
-# who may make mistakes occasionally.
-RESERVED_WORDS = frozenset(itertools.chain(keyword.kwlist, dir(__builtins__)))
-
-TEMPLATE_NAME = 'samplegen_template.j2'
+FILE_HEADER = {"copyrightLines": ["Copyright (C) 2019  Google LLC"],
+               "licenseLines": ["Licensed under the Apache License, Version 2.0 (the \"License\");",
```

Comment:
Binding the copyright and license to the template feels wrong. Using literals in the python code is marginally better because it leaves the door open to obtaining them from input files.

+1 to multiline string and `split("\n")`.

Reply:
Went with splitting lines in the template.

Reply:
OK, sounds good.

Addressing your previous comment above: IMO it's marginally easier to change the license+copyright in a template file than in a code file.